_po/ja/how-to/laravel.po

```diff
@@ -420,12 +616,21 @@ msgid ""
 ") // Add\n"
 "    ->name('posts.search'); // Add"
 msgstr ""
+"Route::get('/',  [\\App\\Http\\Controllers\\PostController::class, 'index']) // Ad"
+"d\n"
+"    ->name('posts.index'); // Add\n"
+"Route::get('/search',  [\\App\\Http\\Controllers\\PostController::class, 'search']"
+") // Add\n"
+"    ->name('posts.search'); // Add"
 
 msgid ""
 "Route::get('/dashboard', function () {\n"
 "    return view('dashboard');\n"
 "})->middleware(['auth', 'verified'])->name('dashboard');"
 msgstr ""
+"Route::get('/dashboard', function () {\n"
+"    return view('dashboard');\n"
+"})->middleware(['auth', 'verified'])->name('dashboard');"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.